README.md

```diff
@@ -626,6 +626,25 @@ An up-to-date list of the convenience operations can be found [here](https://doc
 | GET    | /v1/contextTypes                                                                     | R |
 | GET    | /v1/contextTypes{typename}                                                           | R ]
 
+#### NGSIv2 
+| Method | Path                                                                                    | Action |
+| ------ |:--------------------------------------------------------------------------------------- | ---:|
+| GET    | /v2                                                                                     | R |
+| GET    | /v2/entities                                                                            | R |
+| GET    | /v2\/entities/{entityId}                                                                | R |
+| POST   | /v2\/entities                                                                           | C |
+| PATCH  | /v2\/entities/{entityId}                                                                | U |
+| POST   | /v2\/entities/{entityId}                                                                | U |
```

Comment:
Maybe an entry for `op=append` or `options=...,append,...` (whatever the correct one would be) is missing in the table?